.github/workflows/test.yml

```diff
@@ -19,12 +19,38 @@ jobs:
 
     - uses: ./.github/actions/cache-asdf
 
+    - name: "Set up Cargo cache"
+      uses: actions/cache@v2
+      with:
+        path: |
+          ~/.cargo/bin/
+          ~/.cargo/registry/index/
+          ~/.cargo/registry/cache/
+          ~/.cargo/git/db/
+          target/
+        key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.toml') }}
+
+    - name: Get pnpm store directory
```

Comment:
Copied from https://github.com/pnpm/action-setup#use-cache-to-reduce-installation-time